nvflare/utils/cli_utils.py

```diff
@@ -160,26 +161,57 @@ def is_dir_empty(path: str):
     return len(targe_dir) == 0
 
 
-def save_config(dst_config, dst_path, to_json=False):
-    fmt = ConfigFormat.JSON if to_json else ConfigFormat.PYHOCON
-    ext = ConfigFormat.extensions(fmt)[0]
-    if dst_path.endswith(ext):
+def hocon_to_string(target_fmt: ConfigFormat, dst_config: ConfigTree):
+    if target_fmt == ConfigFormat.JSON:
+        return HOCONConverter.to_json(dst_config)
+    elif target_fmt == ConfigFormat.PYHOCON:
+        return HOCONConverter.to_hocon(dst_config)
+    elif target_fmt == ConfigFormat.OMEGACONF:
+        from nvflare.fuel_opt.utils.omegaconf_loader import OmegaConfLoader
```

Comment:
Not sure if the import statement should be here.  The PyhoconConfig is at the top.